dist/openshift/cincinnati.yaml

```diff
@@ -90,7 +90,11 @@ objects:
                 timeoutSeconds: 3
               resources:
                 limits:
+                  cpu: ${CPU_LIMIT}
                   memory: ${MEMORY_LIMIT}
+                requests:
+                  cpu: "350m"
```

Comment:
Why do we have parameterized limits but hardcoded requests? Would it make sense to have parameters for both instead? 

I would also suggest we don't lock ourselves to always having the same CPU and Memory requirements on Policy Engine and Graph builder, so you can perhaps split this as: 
`PE_CPU_LIMIT` `GB_CPU_LIMIT` and so on...

Reply:
So the suggestion would introduce the following parameters:
 * `GB_CPU_LIMIT`
 * `GB_MEMORY_LIMIT`
 * `PE_CPU_REQUEST`
 * `PE_MEMORY_REQUEST`
correct?

Alternatively, what about going the other direction and hardcoding everything? We are anyway doing devops-style and keeping all parameters in this file, so we may as well remove one level of indirection, I guess? Or do we use/rely on these being parameters, somewhere else?

Reply:
Yes, and we can set the default parameter values in the file (in case someone wants to not use it) 

And then we override them here: https://gitlab.cee.redhat.com/service/saas-cincinnati/blob/master/cincinnati-services/cincinnati.yaml

Like so: https://gitlab.cee.redhat.com/service/saas-telemeter/blob/master/telemeter-services/observatorium.yaml